modules/installation-vsphere-installer-infra-requirements.adoc

```diff
@@ -7,36 +7,81 @@
 [id="installation-vsphere-installer-infra-requirements_{context}"]
 = vCenter requirements
 
-Before you install an {product-title} cluster on your vCenter that uses infrastructure that the installer provisions, you must prepare your environment.
+Before you install an {product-title} cluster on your vCenter that uses installer-provisioned infrastructure, you must prepare your environment. Logging in to a user account or group that has administrative privileges is the simplest way to access all of the necessary permissions in vCenter.
 
 [discrete]
 [id="installation-vsphere-installer-infra-requirements-account_{context}"]
-== Required vCenter account privileges
-
-To install an {product-title} cluster in a vCenter, the installation program requires access to an account with privileges to read and create the required resources. Using an account that has administrative privileges is the simplest way to access all of the necessary permissions.
-
-A user requires the following privileges to install an {product-title} cluster:
-
-* Datastore
-** *Allocate space*
-** *Browse datastore*
-** *Low level file operations*
-** *Remove file*
-* Folder
-** *Create folder*
-** *Delete folder*
-* vSphere Tagging
-** All privileges
-* Network
-** *Assign network*
-* Resource
-** *Assign virtual machine to resource pool*
-* Profile-driven storage
-** All privileges
-* vApp
-** All privileges
-* Virtual machine
-** All privileges
+== Applying the required vCenter account privileges
+
+A vCenter group or user configured in {product-title} must have `read-only` permission, at minimum, for both vCenter and for vCenter entities.
+
+To install an {product-title} cluster in a vCenter, the installation program requires access to an account with privileges to read and create the required resources.
+
+.Procedure
+. Create a new role in vCenter, if you have not already done so.
+
+. As a cluster administrator, apply the following privileges so that a user or group associated with the role can install an {product-title} cluster:
++
+.Required vCenter privileges
+[cols="1a,5a",options="header"]
+|===
+
+|Entity
+|Privileges
+
+|Datastore
+|* *Allocate space*
+* *Browse datastore*
+* *Low level file operations*
+* *Remove file*
+
+|Folder
+|* *Create folder*
+* *Delete folder*
+
+|vSphere Tagging
+|* All privileges
+
+|Network
+|* *Assign network*
+
+|Resource
+|* *Assign virtual machine to resource pool*
+
+|Profile-driven storage
+|* All privileges
+
+|Tasks
+|* All privileges
+
+|vApp
+|* All privileges
+
+|Virtual machine
+|* All privileges
+
+|===
+
+[start=3]
+. Apply `read-only` permissions to the parent entitities of the node VMs and to the vCenter itself, as outlined in the following table:
```

Comment:
`Read-Only` is a role not a permission in vcenter.

Reply:
> `Read-Only` is a role not a permission in vcenter.

`Read` is the permission right? So it should say "`Read` permissions" not `Read-only` role, right?

Reply:
It should say - " Apply `Read-Only` role to the parent entities of the node VMs and to the vCenter itself, as outlined in the following table:"

Reply:
also typo in entities I think.

Reply:
Ah, OK. I updated wording to follow what's in GH vcenter doc: `Read` permission

Reply:
OK, I was going to update to @patrickdillon's suggestion based on similar wording in the [vCenter GH doc](https://vmware.github.io/vsphere-storage-for-kubernetes/documentation/vcp-roles.html#dynamic-provisioning). But I've gone with Hemant's suggestion here, and changed to, "Apply the `Read-only` role.

Reply:
Also, I removed the GH link because IMO the important information in that doc is now in this one, including the `read-only` dynamic provisioning table. 

@gnufied and @patrickdillon As a whole, does this PR look good to you now?

cc: @jcpowermac